### PR TITLE
Adds `thing_id` and additional client ID

### LINKED ISSUE #309: Add `thing_id` and additional client ID to EEPROM
EEPROM currently only has `client_id` but it needs two client IDs: `client_id_config` and `client_id_data`. EEPROM also needs `thing_id`.

### PR DESCRIPTION
**Pull request by jarell-santella**
Adds `thing_id` as well as adds an additional client ID by separating the current `client_id` into two:
1. `client_id_config`
2. `client_id_data`

Closes #309

### CODE REVIEW

**Review by farzadpanahi — APPROVED** [review]

**Review by sjpark608 — APPROVED** [review]
looks good

**Review by farzadpanahi — APPROVED** [review]

File: src/test_edgepi/unit_tests/test_calibration/test_access_eeprom.py [hunk 1]
```diff
@@ -132,52 +132,52 @@ def test_get_message_of_interest(mocker, msg, eeprom):
...
+                  2:CalibParam(gain = 0.9952694074595573, offset= 0.022449294616625794),
+                  3:CalibParam(gain = 0.9930298122067595, offset= 0.052559933743999566),
+                  4:CalibParam(gain = 0.9986894662368055, offset= -0.01639367419587856),
+                  5:CalibParam(gain = 0.9962817511793706, offset= 0.012169769827208299),
+                  6:CalibParam(gain = 0.9977479673435563, offset= -0.006497032357507848),
+                  7:CalibParam(gain = 0.9967431970740426, offset= 0.005837911244602978),
+                  8:CalibParam(gain = 1.0093016097792906, offset= -0.05178411804847496),
+                  9:CalibParam(gain = 0.9977895773887948, offset= -0.030203102327682706),
+                  10:CalibParam(gain = 1.0011882603521973, offset= -0.02699318647809985),
+                  11:CalibParam(gain = 0.9988614798071136, offset= -0.010834493145652147)}
```
Comment (naomiven) [inline src/test_edgepi/unit_tests/test_calibration/test_access_eeprom.py:155]: are these randomly generated inputs? why not use lib random?

**Review by naomiven — COMMENTED** [review]
looks good. i'm not sure how you are testing the `CalibParam` but if you find yourself adding a lot of random inputs it's a good idea to use a lib for that like `random`. not a big deal though

File: src/test_edgepi/unit_tests/test_calibration/test_access_eeprom.py (hunk 1 above)
Comment (sjpark608) [reply src/test_edgepi/unit_tests/test_calibration/test_access_eeprom.py:155]: these are not randomly generated inputs. These values are compared with a dummy serialized protobuf value residing in SDK for testing purpose.

**Review by sjpark608 — APPROVED** [review]

**Review by naomiven — APPROVED** [review]

**Pull request merged**